src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -256,7 +256,7 @@ export default class Blockchain extends Emittery.Typed<
         let firstBlockTime: number;
         if (options.chain.time != null) {
           // If we were given a timestamp, use it instead of the `_currentTime`
-          const t = +options.chain.time;
+          const t = options.chain.time.getTime();
```

Comment:
I changed this to `.getTime()` because I changed the type of `chain.time` to `Date` instead of `Date | number`. I'm not sure why there was a `| number`. It's not in either legacy documentation that a user can provide a UTC millisecond timestamp; both CLI and Core documentation states it should be a ISO 8601 string or a Date object (core only)

I also double checked, and the `ganache-core@2` does assume it's a Date:
- https://github.com/trufflesuite/ganache-core/blob/develop/lib/blockchain_double.js#L88-L90
- https://github.com/trufflesuite/ganache-core/blob/develop/lib/blockchain_double.js#L1251-L1255

Reply:
good catch